src/OpenSage.Mods.Generals/Gui/GameOptionsUtil.cs

```diff
@@ -1,4 +1,4 @@
-﻿using System;
```

Comment:
This file has changes because I was working off of a commit prior to the latest on master but which had some other changes that have already been merged to master. Should be inconsequential but I can cherry-pick the commits over to a new branch if necessary.